packages/desktop/src/services/PersistentConfigServiceElectron.ts

```diff
@@ -10,9 +9,7 @@ import { find } from "lodash";
 
 const OPTIONS: Options<Record<string, unknown>> = {
     // Defines a validation schema for data inserted into the persistent storage
-    // if a breaking change is made see migration patterns in elecron-store docs
     schema: {
-        // Deprecated in 6.0.0
```

Comment:
Were these comments removed intentionally?

Reply:
nope! added back in [38e7ac2](https://github.com/AllenInstitute/biofile-finder/pull/223/commits/38e7ac251efb44254d2eb9eee39389e2b01ecc1b)